### ISSUE TITLE
Missing fantasy consoles

### ISSUE THREAD

**Issue by cosmo0** [paladin-t/fantasy#66]
Hello

You already listed Octo wich is more an IDE than a fantasy console. The "actual" underlying fantasy console is CHIP-8, and a lot of resources can be found here: https://chip-8.github.io/ (Octo is pretty cool and should stay on the list).

There is also Zany80, which is based around the Z80 processor : https://pixelhero.dev/zany80/ ; it's rough and hasn't been updated in ages (though not more than other projects on the list), but I think it's interesting that it's based on actual hardware, both retro (used in many 80s computers) and current (still used in a lot of things).

**Comment by paladin-t** [comment]
Thanks for the info.

The maintainer's intention is also considered. Zany80 was on the list and has been [removed](https://github.com/paladin-t/fantasy/pull/48) by the author.

CHIP-8 is at the edge. It describes itself a programming language, sometimes a classic computer (which is very close to an FC I think). It is also often used to guide designing actual hardwares, I prefer to name it a VM (i.e. JVM), or independent architecture (i.e. RISK-V).

**Comment by cosmo0** [comment]
Too bad for Zany80.

Maybe you could list "retired" projects? A lot of the projects in the list haven't been updated in several years and, from the comments, don't work at all. I could help you with that.

CHIP-8 is described everywhere I see exactly like a fantasy console:

https://chip-8.github.io/links/

> Created in 1977, CHIP-8 is the original fantasy console.

https://en.wikipedia.org/wiki/CHIP-8

> CHIP-8 programs are run on a CHIP-8 virtual machine. It was made to allow video games to be more easily programmed for these computers.

I think it'd be interesting to at least mention its existence and legacy?

**Comment by paladin-t** [comment]
> Maybe you could list "retired" projects?

Added a [retired](https://github.com/paladin-t/fantasy/wiki/Retired) wiki page. Feel free to edit it.

Also feel free to mention if any of them on the main list is not working at all.

> CHIP-8 is described everywhere I see exactly like a fantasy console

I see. But the [official](https://chip-8.com/) site only provides instructions to build virtual/actual CHIP-8 computers. I'd say ok to add it if there were official downloadable CHIP-8 interpreter. Some other softwares are implementing CHIP-8 compatibles, but they are just referring it as an open standard. Correct me if I'm missing anything here, but I'd be glad to mention its existence.

**Comment by cosmo0** [comment]
Thanks for the wiki page :)

I'll try to check the list and see if any stand out as "not working" (I can't try them all).

> But the official site only provides instructions to build virtual/actual CHIP-8 computers

That's because it's not an official site, it's just a regular store selling stuff under the CHIP8 name ; there aren't even legal infos of the company. CHIP-8 is indeed an open standard ; the link I posted earlier ( https://chip-8.github.io/ ) is the best one I found for the documentation.

There's also VDP16, trying to simulate a 16bits console: https://github.com/Brunni132/vdp16-samples/tree/gh-pages
